examples/llm/src/models/utils/make_llama_work_with_autoclass.py

```diff
@@ -0,0 +1,329 @@
+# Copyright 2022 MosaicML Examples authors
```

Comment:
I didn't really review this file, assuming it is copied

Reply:
there are ~10 lines at the end and `class C4SentencePieceConfig(LlamaConfig):` that are not copied